mos.yml

```diff
@@ -132,6 +136,7 @@ libs:
   - location: https://github.com/mongoose-os-libs/file-logger
   - location: https://github.com/mongoose-os-libs/homekit-adk
   - location: https://github.com/mongoose-os-libs/http-server
+  - location: https://github.com/mongoose-os-libs/onewire
```

Comment:
since this is currently only used by 1 and 1PM, we should onyl add this dependency to them.